system/datatypes/mailinglist.php

```diff
@@ -148,10 +148,10 @@ public function indexAction() {
 					// send email so that action can be confirmed
 					$link = $this->constructFullPath(sprintf('%s/confirm?code=%s', urlencode($this->pagename), urlencode($code)));
 					$confirmTxt = __('ml-please-confirm-email');
```

Comment:
Is this `$confirmTxt` still used anywhere? If not, the corresponding string should probably be removed too.

Reply:
Yup, still used in the e-mail body.